.github/workflows/release.yaml

```diff
@@ -71,6 +71,7 @@ jobs:
         env:
           TAG: ${{ github.ref_name}}
         run: |
+          set +e
```

Comment:
Maybe add a comment to why this is needed, otherwise we might remove it again in the future :slightly_smiling_face: